src/main/java/cart/exception/AuthException.java

```diff
@@ -0,0 +1,8 @@
+package cart.exception;
+
+public class AuthException extends RuntimeException{
```

Comment:
1. 전반적으로 띄어쓰기 체크해주세요!
    ```suggestion
    public class AuthException extends RuntimeException {
    ```
2. 전반적으로 개행 체크해주세요.
    - 의미 없는 불필요한 두 줄 개행
    - 상수와 멤버변수 사이 개행

Reply:
넵 확인하겠습니다 !